pkg/rclone/rclone.go

```diff
@@ -216,7 +214,13 @@ func (r Rclone) Unmount(ctx context.Context, volumeId string, targetPath string)
 		return err
 	}
 	if resp.StatusCode != 200 {
-		return fmt.Errorf("unmounting failed: couldn't delete mount: %s", string(body))
+		var result map[string]interface{}
+		json.Unmarshal(body, &result)
+		val, ok := result["error"]
+		if !ok || val != "mount not found" {
```

Comment:
```suggestion
		errorMsg, found := result["error"]
		if !found || errorMsg != "mount not found" {
```

Not a big deal but it is nicer if the names were more descriptive.